content/about/notes/1.1/index.md

```diff
@@ -49,10 +49,10 @@ Istio 1.1.
 
 ## Traffic management
 
-- **New [`Sidecar`](/docs/concepts/traffic-management/#sidecars) Resource**. Enables more fine-grained
-  control over the behavior of the sidecar proxies attached to workloads within a namespace. In
-  particular adds support to limit the set of services a sidecar will send traffic to. This
-  reduces the amount of configuration computed and transmitted to
+- **New `Sidecar` Resoruce**. The new [sidecar](/docs/concepts/traffic-management/#sidecars) resource
```

Comment:
I'm not sure if I mispelled this, but should be `Resource` rather than `Resoruce`.